anime_downloader/sites/kissanimex.py

```diff
@@ -0,0 +1,41 @@
+from anime_downloader.sites import helpers
+from anime_downloader.sites.anime import Anime, AnimeEpisode, SearchResult
+
+class KissAnimeX(Anime, sitename = 'kissanimex'):
+    sitename = "kissanimex"
+
+    @classmethod
+    def search(cls, query):
+        url = 'https://kissanimex.com/search?url=search'
+        r = helpers.get(url, params={'q': query})
+        soup = helpers.soupify(r.text)
+        items = soup.find_all('td')
+        search_results = [
+                SearchResult(
+                    title = x.find('a').text,
+                    url = 'https://kissanimex.com' + x.find('a')['href']
+                    )
+                for x in items
+                ]
+        return search_results
+
+    def _scrape_episodes(self):
+        r = helpers.get(self.url).text
+        eps = helpers.soupify(r).find('div', id='episodes-sub').select('td > a')
+        episodes = ['https://kissanimex.com' + x.get('href') for x in eps][::-1]
+        return episodes
+
+    def _scrape_metadata(self):
+        self.title = helpers.soupify(helpers.get(self.url).text).find('a', class_='bigChar').text
```

Comment:
classes can be .class
```suggestion
        self.title = helpers.soupify(helpers.get(self.url).text).find('a.bigChar').text
```